### PR TITLE
Update FAQ: Ensure security on external links

### LINKED ISSUE #3031: Missing relationship attributes on external links
## Where to find the issue

FAQ

## Problem and suggested change

This issue was brought up by @MikeMcC399 in recently submitted PR [#3030](https://github.com/corona-warn-app/cwa-website/pull/3030#issuecomment-1185457029)

There has never been a written policy here about external links but for consistency `rel='noopener noreferrer'` would need to be added to ensure security for external links, such as

`<a href='https://tools.rki.de/PLZTool/en-GB' target='_blank'>Robert Koch Institute Tool: Health authority by postal code or place</a>`

---
Internal Tracking ID: [EXPOSUREAPP-13638](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13638)

### PR DESCRIPTION
**Pull request by larswmh**
Fixes #3031 

`rel='noopener noreferrer'` was added to all external links in the FAQ. 

Some links that were not covered by #3030 due to the fact that they had `target='_self'` in it were corrected to `target='_blank'`, so that they open up in a new tab as intended.

---
Internal Tracking ID: [EXPOSUREAPP-13638](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13638)

### CODE REVIEW

**Review by dsarkar — APPROVED** [review]

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
rel='noopener' should be removed for internal links e.g.
https://github.com/corona-warn-app/cwa-website/blob/0d8dba17393bbd56e2c5dcd267737959a7b78d46/src/data/faq.json#L416

File: src/data/faq_de.json [hunk 1]
```diff
@@ -2042,7 +2042,7 @@
...
                                     "<b>AKTUELL</b><br/> Das Problem ist in Version 1.7.1 behoben worden.<hr/>",
-                                    "Die Meldung <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/1497' target='_blank'>'ExposureDetectionIsAlreadyRunning'</a> <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/1512#issuecomment-727206633' target='_blank'>kann ignoriert werden</a>. Die CWA funktioniert korrekt und dieses Problem wird in einem <a href='https://github.com/corona-warn-app/cwa-app-ios/pull/1510' target='_blank'> bevorstehenden Update korrigiert</a>."
+                                    "Die Meldung <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/1497' target='_blank'>'ExposureDetectionIsAlreadyRunning'</a> <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/1512#issuecomment-727206633' target='_blank' rel='noopener noreferrer'>kann ignoriert werden</a>. Die CWA funktioniert korrekt und dieses Problem wird in einem <a href='https://github.com/corona-warn-app/cwa-app-ios/pull/1510' target='_blank' rel='noopener noreferrer'> bevorstehenden Update korrigiert</a>."
```
Comment (MikeMcC399) [inline src/data/faq_de.json:2045]: The first link (to https://github.com/corona-warn-app/cwa-app-ios/issues/1497) in this list is not fixed.

**Comment by larswmh** [comment]
@MikeMcC399 any `rel=' '` has now been removed from internal links, thanks for the suggestion

File: src/data/faq_de.json [hunk 2]
```diff
@@ -2003,7 +2003,7 @@
...
-                                    "Benutzt Ihr Netzwerk bzw. Ihr Gerät ein Firewall, überprüfen Sie die Erreichbarkeit der Domänen, die in <a href='#access-list' target='_blank' rel='noopener'>[Google/Android]: Firewall/Router Konfiguration: URSACHE: 4000, error during web request, HTTP status 901</a> aufgelistet sind.",
+                                    "Benutzt Ihr Netzwerk bzw. Ihr Gerät ein Firewall, überprüfen Sie die Erreichbarkeit der Domänen, die in <a href='#access-list' target='_blank'>[Google/Android]: Firewall/Router Konfiguration: URSACHE: 4000, error during web request, HTTP status 901</a> aufgelistet sind.",
                                     "Sollten diese Schritte das Problem nicht beheben, schreiben Sie bitte einen Kommentar hier: <a href='https://github.com/corona-warn-app/cwa-app-android/issues/968' target='_blank' rel='noopener'>cwa-app-android/issues/968</a>."
```
Comment (MikeMcC399) [inline src/data/faq_de.json:2007]: Should be rel='noopener noreferrer'

File: src/data/faq_de.json [hunk 3]
```diff
@@ -2220,8 +2220,8 @@
                                 "active": false,
                                 "textblock": [
                                     "<b>AKTUELL 14. Januar 2021</b><br/>Sollten Sie dieses Problem haben, schreiben Sie bitte Ihren Bericht hier: <a href='https://github.com/corona-warn-app/cwa-app-android/issues/1962' target='_blank' rel='noopener'>cwa-app-android/issues/1962</a>.<hr/>",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:2222]: Should be rel='noopener noreferrer'

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
There are still some 'noopener' for external links which need to be converted to 'noopener noreferrer'

File: src/data/faq_de.json (hunk 3 above)
Comment (larswmh) [reply src/data/faq_de.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/3032/commits/2180bce04f3ca6fcfd3f0e9d26b27cae68ebebe1, thanks!

File: src/data/faq_de.json (hunk 2 above)
Comment (larswmh) [reply src/data/faq_de.json]: Fixed in https://github.com/corona-warn-app/cwa-website/pull/3032/commits/2180bce04f3ca6fcfd3f0e9d26b27cae68ebebe1, thanks

**Comment by MikeMcC399** [comment]
@larswmh 

Please check also for incorrectly written `target='blank'` which should be `_blank` + `rel` attributes.

(I'm in the process of trying to write a Cypress test, so I'm not finding all the issues at once.)

File: src/data/faq_de.json [hunk 4]
```diff
@@ -3086,7 +3086,7 @@
...
+                    "description": "<p>Antigen-Schnelltests ähneln von der Vorgehensweise dem <a href='#glossary_selftest' target='_blank'>Selbsttest</a>, werden allerdings von geschultem Personal durchgeführt. Eine Gegenüberstellung der beiden Antigentest-Typen wurde vom RKI bereit gestellt: <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Downloads/Flyer-Antigentests.pdf?__blob=publicationFile' target='_blank'>Link zur Gegenüberstellung vom RKI</a></p><p>Nicht alle Antigen-Schnelltests sind ähnlich zuverlässig. Aus diesem Grund hat das Paul-Ehrlich-Institut sog. Mindestkriterien an Antigen-Schnelltests veröffentlicht: <a href='https://www.pei.de/SharedDocs/Downloads/DE/newsroom/dossiers/mindestkriterien-sars-cov-2-antigentests.pdf?__blob=publicationFile&v=9' target='_blank' rel='noopener noreferrer'>Link zu den Mindestkriterien</a></p><p>Ein positives Ergebnis eines Antigen-Schnelltests ist noch keine finale Diagnose und sollte durch einen <a href='#glossary_pcr_test' target='_blank'>PCR-Test</a> bestätigt werden.</p>"
```
[the remark this answers is not in the document]
Comment (larswmh) [reply src/data/faq_de.json]: Resolved by your PR, thanks again!

**Review by MikeMcC399 — APPROVED** [review]
@larswmh 

I'm not detecting any more link issues, so I think you can go ahead and merge.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks a lot for reviewing !

**Pull request merged**